### ISSUE TITLE
Matsu Graphics Patch To-Do List

### ISSUE THREAD

**Issue by orian34** [07th-mod/matsuribayashi#6]
- [x] Start scenario CG need updated ones.
- [ ] Fix kakera effect? Used in fragment 41, 43, 45, 46, 47, 48, 49, 51. (@idealpersona)
- [x] Added se/bgm from previous chapters: wa_003, msys06, msys32.
- [ ] Kakera requirements need a translation checking. eg kakera 14 needs 1 but doesn't show it. Confusing requirements.
- [x] Okonogi name tag is white(needs to be green). 
- [x] Egret to Heron. Feng to Phoenix. (needs to be checked and changed if nametags use wrong one, I fixed some manually but might have missed a few).
- [x] CGs need adaptation, waku_b, waku_w, white_mono1, etc, overall new CG upgrade.
- [x] UI file for all the menu stuff.
- [x] Add cubemap for OG(done, needs to be added, check discord?)
- [x] Make hanyu filter CG more gray.
- [x] Typo in si_matsuri_02 "hapier". Maybe double check text CG spellings?
- [x] (optional)Need transparent text cgs for Maid In Heaven and I can whip up an upgraded scene.

**Comment by enumag** [comment]
- [ ] Voice pack
- [ ] Sprite pack
- [ ] Ryukishi sprites

**Comment by idealpersona** [comment]
Kakera effect is a problem in the latest update of the vanilla game and has been reported to Mangagamer.

Also reported the GOG version being out of date and O[A] also reported this to doddler directly.

I think there's nothing else pending from me now besides this so I'm gonna unassign myself until we get word back - if anything else needs my attention feel free to assign me back.

**Comment by drojf** [comment]
I've finished checking the ryukishi sprite mapping for enumag. Enumag - afaik the sprite list you gave me was completely covered (except for new characters below), but you may want to double check that.

- There are two characters without ryukishi sprites
    - `ama_` (amakusa)
    - `miyo1_` / `miyo3_` (miyo)

- There are two mappings which are only used as ps3 portrait images (zoomed in versions). Due to how my script works, they still are listed as `sprite/name` rather than `portrait/name` in the .csv file:
    - `aks2_kakusei_` from `_mats_021.txt`
    - `ama_odoroki_` from `_mats_022.txt`
    - For `aks2_kakusei_`, I just picked the equivalent non-zoomed in sprite. For `ama_odoroki_` (and all of ama_), since there is no ryukishi sprite, it's just set to NEW_CHAR_AMA

The latest .csv can be downloaded here: https://github.com/drojf/match_higurashi_scripts/blob/master/imageComparer/noconsole_output.txt.csv . The new matches start at `sprite/aks1_sakebi_`, or Exel row 440

**Comment by ItaloKnox** [comment]
It took me a while but the disclaimer is here. We still need the japanese version (maybe from @Norgus?).

I also created a disclaimer for saves since this seems to happen very often, this one can be included at booting the game or after you click start. I think it works better when booting, but I'm not very satisfied with the design. It should fit nicely with the voices/shortcuts screen when pressing *start*, though. Of course, this one is just an extra. Be free to stash it.

![voices](https://user-images.githubusercontent.com/4702556/85352634-c8081880-b4dc-11ea-80e7-6e8edaecf531.png)
![saves](https://user-images.githubusercontent.com/4702556/85352637-c9394580-b4dc-11ea-8d18-53c84fb3fa95.png)

**Comment by Norgus** [comment]
I don't mind, but is the 'Chapter Jump' feature really after starting a new game? I checked in onikakushi (just quickly available on my system) and the feature seemed to be within the Extras menu.

I also can't remember off-hand what we referred to the censorship and voice matching as in Japanese and I don't want to use inconsistent terminology - any ideas where I can check to see what it was referred to before?
